src/providers/Theme.vue

```diff
@@ -43,6 +43,22 @@ export default defineComponent({
   --color-5: #fff;
   --bg-1: #002555; // Widget background
 
+  &[data-lang='au'] {
+    --color-0: #002555;
+    --color-1: #002555;
+    --color-1-contrast: #002555;
+    --color-2: #ecb250; // Web / Tile / Button / Secondary
+    --bg-1: #ffffff; // Widget background
+  }
+
+  &[data-lang='au'][data-dark='true'] {
+    --color-0: #002555;
+    --color-1: #ecb250;
+    --color-1-contrast: #fff;
+    --color-2: #ecb250; // Web / Tile / Button / Secondary
+    --bg-1: #002555; // Widget background
+  }
+
   &[data-lang='nz'],
   &[data-theme='qmc'] {
     --color-0: #000000;
```

Comment:
Important bit here ☝️ 
Wondering if there is a better way to implement this.

Reply:
This looks fine from what I can tell.. What's your concern?

Reply:
I don't think i needed a `&[data-lang='au']` (line 46 to 52). Instead, it should have been the default.